doc/adr/0002-summary-of-decisions-thus-far.md

```diff
@@ -0,0 +1,85 @@
+# 2. Summary of decisions thus far
+
+Date: 2024-10-09
+
+## Status
+
+Accepted
+
+## Context
+
+This is a high-level summary of previous decisions that have been made on this
+project. ADRs will be used for all new architectural decisions.
+
+## Decision
+
+Below is a list with the specific explanations on the architectural decisions
+that have been made thus far in this project.
+
+#### Use Django and Django Admin.
+
+Previously, a Web 1.0 Python application was being built to create this
+microservice. It was felt that utilizing Django and the built in Admin service
+could easily support the projected user base size of the Affiliations
+Microservice and it would increase the speed of delivery on this project. It was
+also a benefit that it was another Python framework and a tool being utilized on
+another project on the team to create cross over skills.
+
+Although it is not recommended to base your website solely on the built-in admin
+service, since this is going to be a small internal tool to replace a
+spreadsheet and JSON files, that will later be mostly handled by API request via
+the GPM we felt that it would serve our purposes; we acknowledged that the
+recommendations and limitations that would be brought on by building our service
+solely on top of the built-in Django Admin service tool.
+
+#### Using PostgresSQL
+
+We were interested in a relational database management system and PostgresSQL
+has many features that are supported by Django. Postgres is also recommended as
+the database back-end by the creators of Django.
+
+#### Using Unfold Admin
+
+After discussion of the base Django Admin UI not aligning with our vision of the
+project, we discussed the making custom CSS vs finding a pre-built theme to
+utilize. With the resources we had available, using a pre-built theme we can
+tweak and customize seemed like a better use of time. After looking at several
+options, we settled on Unfold Admin not only because of the styling, but because
+it was well supported and the community seemed active surrounding it.
+
+#### Using Custom JS
+
+Our goal in this project is to not make it too overly complex. It is meant to be
```

Comment:
Our goal in this project is to not make it ~~too~~ overly complex.

Reply:
Made this change!